skimage/exposure/tests/test_exposure.py

```diff
@@ -212,12 +212,12 @@ def test_adapthist_grayscale():
     img = rgb2gray(img)
     img = np.dstack((img, img, img))
     with expected_warnings(['precision loss|non-contiguous input']):
-        adapted = exposure.equalize_adapthist(img, 10, 9, clip_limit=0.01,
+        adapted = exposure.equalize_adapthist(img, (52, 64), clip_limit=0.01,
```

Comment:
Are the old results unobtainable with the new method? My understanding was that this was backwards compatible with appropriate settings. I'd feel better retaining the old test and adding a new test case, if possible.